src/components/Accordion/Accordion.stories.tsx

```diff
@@ -1,4 +1,4 @@
-import type { Meta, StoryFn } from '@storybook/react';
+import type { Meta, Story } from '@storybook/react/types-6-0';
```

Comment:
Please restore this change, see https://github.com/storybookjs/storybook/issues/14190